update.sh

```diff
@@ -5,6 +5,11 @@ set -ex
 
 cd `dirname $0`
 
+if [[ ! $VIRTUAL_ENV ]]
+then
+	user_option='--user'
```

Comment:
Maybe add a comment like "sandbox the install to the virtualenv or, if not available, to the user"?